web/client/translations/data.de-DE

```diff
@@ -1305,10 +1305,10 @@
             },
             "aggregationAttribute": {
                 "line": "Y-Attribut",
-                "pie": "Wert",
```

Comment:
For the other languages, you can use the english translations.
Please add/change them